book_maker/translator/chatgptapi_translator.py

```diff
@@ -90,5 +120,110 @@ def translate(self, text):
             t_text = self.get_translation(text)
 
         # todo: Determine whether to print according to the cli option
-        print(t_text.strip())
+        if needprint:
+            print(re.sub("\n{3,}", "\n\n", t_text))
+
+        elapsed_time = time.time() - start_time
+        print(f"translation time: {elapsed_time:.1f}s")
```

Comment:
There are some `print` statements that are used for debugging purposes, but I agree with your point that the `logging` module can be used instead. Users are primarily interested in the original text, translation, and progress, which are the original output.
What's your opinion? @yihong0618

Reply:
I made some modifications and refactoring. Now the only thing that changes when not using `--accumulated_num` is the output of the translation time. How about commenting out these outputs and just merge? I see other people want to refactor the code @yihong0618